yazi-config/preset/keymap.toml

```diff
@@ -151,8 +151,9 @@ keymap = [
 	{ on = [ "g", "<Space>" ], run = "cd --interactive", desc = "Go to a directory interactively" },
 
 	# Help
-	{ on = "~",    run = "help", desc = "Open help" },
-	{ on = "<F1>", run = "help", desc = "Open help" },
+	{ on = "~",		run = "help", 	desc = "Open help" },
+	{ on = "<F1>",	run = "help", 	desc = "Open help" },
+	{ on = "F",		run = "filter", desc = "Apply a filter for the help items" },
```

Comment:
https://github.com/sxyazi/yazi/blob/f024ce03e7b97f55f3bfad9da4b33458cea765d5/yazi-config/preset/keymap.toml#L101-L102

It's already in `[manager]` section, https://yazi-rs.github.io/docs/quick-start#filter-files

Reply:
This is filter for file list. I'm talking about interactive filtering of entries in help listing

Reply:
> in help listing

So why did you add it to `[manager]` rather than `[help]`?